### PR TITLE
DOCSP-41826 - Remove MDB 3.6

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41826
Staging:
- [What's New](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-41826-server-eol/whats-new/)
- [Upgrade](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-41826-server-eol/upgrade/#version-3.0-potential-breaking-change)
- [Compatibility](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-41826-server-eol/compatibility/)
- [Authentication](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-41826-server-eol/fundamentals/authentication/#default)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -32,6 +32,16 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _csharp-version-3.0:
```
Comment (jordan-smith721) [inline source/whats-new.txt:35]: Can you add a link to this at the top, under the "Learn what's new in:" section?

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ one suggestion

File: source/whats-new.txt (hunk 1 above)
Comment (mongoKart) [reply source/whats-new.txt:35]: ah good call

**Review by adelinowona — APPROVED** [review]
LGTM!

**Pull request merged**